### PR TITLE
Review & edit AD direct integration ext auth user story

### PR DESCRIPTION
**Pull request by asteflova**
A follow-up on https://github.com/theforeman/foreman-documentation/pull/2938: The goal is to cover the AD user story from beginning to end and make sure it's easy to read and follow.

Also, a follow-up on https://github.com/theforeman/foreman-documentation/pull/2737 which I'm taking over from @adamlazik1.

* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
 
 .Procedure
+On your {ProjectServer}:
+
 . Install the required packages:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation oddjob oddjob-mkhomedir realmd samba-common-tools sssd
```
Comment (asteflova) [inline guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc:24]: Confirmed in https://issues.redhat.com/browse/SAT-26355 and in [RHEL AD integration docs](https://issues.redhat.com/browse/SAT-26355).

**Comment by asteflova** [comment]
While reviewing the existing docs with SSSD Engineering, we noticed that the existing procedures use both Samba and SSSD to integrate with AD. That’s something that deserves a closer look because SSSD "should" be enough to perform the join and it seems impractical to use both.

This is why I’m going to pause my efforts in reviewing the whole scenario and close this PR. For the immediate future, I’ll only focus on fixing bugs that we currently track for the existing procedures and also on minor doc improvements, such as introductions. I’ll open a new PR for that. I want to keep this one as is for future reference.

**Comment by ekohl** [comment]
> This is why I’m going to pause my efforts in reviewing the whole scenario and close this PR. For the immediate future, I’ll only focus on fixing bugs that we currently track for the existing procedures and also on minor doc improvements, such as introductions. I’ll open a new PR for that. I want to keep this one as is for future reference.

Should we open an issue to be reminded of it?

**Comment by asteflova** [comment]
> Should we open an issue to be reminded of it?

I have one downstream: https://issues.redhat.com/browse/SAT-26039  Let me know if that's not enough.

**Pull request closed without merging**